ceph.spec.in

```diff
@@ -512,7 +519,14 @@ export RPM_OPT_FLAGS=`echo $RPM_OPT_FLAGS | sed -e 's/i386/i486/'`
 %endif
 		$MY_CONF_OPT \
 		%{?_with_ocf} \
-		%{?_with_tcmalloc} \
+%if 0%{with tcmalloc}
+		--with-tcmalloc \
+		--without-jemalloc \
+%endif
+%if 0%{with jemalloc}
+		--without-tcmalloc \
+		--with-jemalloc \
+%endif
```

Comment:
Should `./configure` bail out if both `--with-tcmalloc` and `--with-jemalloc` are specified?

Or maybe we need to have "`--with-malloc=<foo>` ?